src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -131,6 +132,23 @@ public String addUserIdToUrl(String link) {
         }
         return result;
     }
+
+    /**
+     * Returns the nationalities as HTML code.
+     */
+    public static List<ElementTag> getNationalitiesAsElementTags(String existingNationality) {
+        List<String> nationalities = NationalityHelper.getNationalities();
+        ArrayList<ElementTag> result = new ArrayList<ElementTag>();
```

Comment:
The left-hand side should be the interface, i.e `List<ElementTag> result = ...`. Look up "Loose coupling" for this.